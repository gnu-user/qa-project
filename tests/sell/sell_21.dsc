TITLE
    Sale Transaction Session Constraint
DESCRIPTION
    Run the login transaction using an admin account. Initiate the sell 
    transaction. Enter a valid title, and enter valid numerical input (a double 
    with two decimal places between 0.00 and 999.99 inclusive) when prompted to 
    enter a ticket price. Enter an integer value between 0 and 100 inclusive 
    when prompted to enter a ticket volume. Attempt to initiate the sell 
    transaction again. Verify that the attempt is rejected and an error occurs.
DEPENDENCIES
    sell_19
