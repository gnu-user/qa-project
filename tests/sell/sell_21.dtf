03_A valid title.______administrator___025_010.00