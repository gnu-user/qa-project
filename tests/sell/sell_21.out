Welcome.
Current user accounts file read successfully.
Available tickets file read successfully.
> 
Enter username: 
Username accepted.
> 
Enter the event title: 
Enter the sale price in dollars: 
Enter the number of tickets for sale: 
Tickets listed for sale successfully.
> 
Invalid transaction, only one sell transaction per session accepted.